openfisca_us/parameters/usda/snap/emergency_allotment/minimum.yaml

```diff
@@ -0,0 +1,10 @@
+description: Minimum amount of SNAP emergency allotment.
+
+CA:
+  2022-01-01: 95
+
+metadata:
+  period: month
+  reference:
+    - title: "California: COVID-19 Waivers & Flexibilities"
+      href: https://foodforpeople.org/covid-19-calfresh
```

Comment:
I can't find a legislative reference for this. It's probably here: https://www.cdss.ca.gov/inforesources/legislation-and-regulations

But two sites have mentioned the $95 minimum.